Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
 import './Card.css';
+
+const click = () => {
+    window.open('https://reactjs.org/', '_blank');
+}
+const CardContent = () => {
+    return (
+        <div className="Card-Content" onClick={click}>
+            <p className="bold">Get started with React</p>
```

Comment:
For something like this, you don't need to make a class and do this in CSS, you can just do inline styles because you are only changing one thing, especially if this is the only thing that will be bold.

`style={{ fontWeight: bold }}`